### PR TITLE
Hw03

### PR DESCRIPTION
**Pull request by soulsword1**
Доброго дня. 
Прохання перевірити домашку

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -0,0 +1 @@
+DB_HOST = mongodb+srv://max:BKsqdstQahhmLK5c@cluster0.gv7ll21.mongodb.net/db-contacts?retryWrites=true&w=majority
```
Comment (ceREAL133) [inline .env:1]: .env file must be in .gitignore. It shouldn't be visible

File: controllers/contacts.js [hunk 2]
```diff
@@ -0,0 +1,109 @@
+const joi = require("joi");
+const httpError = require("../utils/HttpError");
+const controlWrapper = require("../utils/ControlWrapper");
+const Contact  = require("../models/contact");
+
+const schemaPost = joi.object({
```
Comment (ceREAL133) [inline controllers/contacts.js]: schema should be in separate file

**Review by ceREAL133 — APPROVED** [review]

File: .env (hunk 1 above)
Comment (soulsword1) [reply .env:1]: added it to gitignore and created server on render.com

File: controllers/contacts.js (hunk 2 above)
Comment (soulsword1) [reply controllers/contacts.js]: added schemas to model file

**Pull request merged**